packages/sdk-components-react-radix/src/__generated__/popover.props.ts

```diff
@@ -17,7 +17,6 @@ export const propsPopoverContent: Record<string, PropMeta> = {
     type: "string",
     defaultValue: "center",
     options: ["center", "start", "end"],
-    description: "Specifies the horizontal alignment of the element.",
```

Comment:
Is this one gonna be correct for example?

Reply:
Previously take from legacy html attribute though not necessary means the same in radix.